Tensile/Tests/yaml_only/test_config.py

```diff
@@ -71,6 +71,9 @@ def configMarks(filepath, rootDir, availableArchs):
         ArchFail = "xfail-%s" % arch
         if markNamed(ArchFail) in marks:
             marks.append(pytest.mark.xfail)
+        ArchSkip = "skip-%s" % arch
+        if markNamed(ArchSkip) in marks:
+            marks.append(pytest.mark.skip)
```

Comment:
New marks should also be added to the list in pytest.ini

Reply:
mark added to pytest.ini